src/main/kotlin/no/nav/melosys/soknadmottak/dokument/DokumentController.kt

```diff
@@ -19,6 +19,7 @@ import org.springframework.web.bind.annotation.RestController
 class DokumentController @Autowired constructor(
     private val dokumentService: DokumentService
 ) {
+    @Deprecated(message = "Pdf som BASE64-string returneres i hentVedlegg")
```

Comment:
`hentVedlegg` er/blir egentlig `hentDokumenter` :  henter alle dokumenter for en gitt søknad, inkludert søknaden. Vi kunne også flytte endepunktet til api/soknader/{soknadID}/dokumenter,

Reply:
Enig :+1: